velox/connectors/Connector.h

```diff
@@ -479,6 +540,28 @@ class Connector {
     return false;
   }
 
+  /// Creates index source for index join lookup.
+  /// @param lookupInputType The input type for lookup request which contains
```

Comment:
'lookup' prefix is redundant

> index columns from index table, and the columns from probe

This is confusing to me. Isn't this a list of probe-side columns that either used in equi-clauses or join condition?